docs-v2/sdks/node.mdx

```diff
@@ -416,3 +417,22 @@ console.log(vars);
 ]
 */
 ```
+
+### Update sync's connection frequency
+
+To update a sync's connection frequency, or revert to the default frequency:
```

Comment:
Same here: "To override a sync's frequency for a specific connection, or revert to the default frequency, us:"